spki/src/fingerprint.rs

```diff
@@ -36,7 +38,6 @@ impl Builder {
 
 impl Writer for Builder {
```

Comment:
Seems like you should be able to completely get rid of `Builder` and this module and update `SubjectPublicKeyInfo::fingerprint_*` to use `DigestWriter`:

https://github.com/RustCrypto/formats/blob/master/spki/src/spki.rs#L63-L86